<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xunluoluxian</class>
 <widget class="QWidget" name="xunluoluxian">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>巡逻路线</string>
  </property>
  <widget class="QPushButton" name="add_xllx">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>20</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加巡逻路线</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>481</width>
     <height>291</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>479</width>
      <height>289</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>路线名称</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>320</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>巡逻点</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="ok_addxllx">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="re_addxllx">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
